packages/page-template-modal/src/components/page-template-modal.js

```diff
@@ -172,13 +113,13 @@ export default class PageTemplateModal extends Component {
 			return;
 		}
 
-		const isHomepageTemplate = find( this.props.templates, { name, category: 'home' } );
+		const template = this.props.templates.find( ( t ) => t.name === name );
 
 		// Load content.
 		const blocks = this.getBlocksForSelection( name );
 
 		// Only overwrite the page title if the template is not one of the Homepage Layouts
-		const title = isHomepageTemplate ? null : this.getTitleByTemplateSlug( name );
+		const title = template.category === 'home' ? null : template.title || '';
```

Comment:
Ok I remember what this was about now. This was replacing a use of `_.find`. `isHomepageTemplate` used to be calculated like this:
```js
const isHomepageTemplate = _.find( this.props.templates, { name, category: 'home' } );
```
So testing `template.category === 'home'` is the correct way to replicate the existing behaviour.

However your point still stands, this doesn't work the way the comment says it will. Templates can have multiple categories, but that's stored in the **`categories`** prop, not **`category`**. None of the current templates have a `category` property, so I think this is already broken.

My guess is we don't want homepage layouts to replace the title because the title is probably already set correctly to the site's actual title. Seems reasonable, so I'll fix/add-this-behaviour. I'll search through all the categories looking for `'home'`.

Reply:
I've fixed the "don't set the title for home layouts" behaviour. Added tests for the behaviour since it seems like the sort of thing that'd be overlooked when future changes are made.